src/solver/conv_hip_implicit_gemm_fwd_xdlops.cpp

```diff
@@ -227,6 +226,42 @@ void RunCKSolution(const Handle& handle,
     }
 }
 
+InvokerFactory
+MakeInvokerFactoryHipImplGemmFwdXdlops(const ProblemDescription& problem,
+                                       const PerformanceConfigHipImplicitGemmFwdXdlops& config)
+{
+
+    auto ck_args               = CKArgs{problem};
+    miopenDataType_t data_type = problem.conv_problem.GetInDataType();
+    auto config_idx            = config.index;
+
+    InvokerFactory ret =
+        [ck_args = std::move(ck_args), data_type, config_idx](const std::vector<Kernel>& kernels) {
+            std::ignore = kernels;
+            return [ck_args, data_type, config_idx](const Handle& handle,
```

Comment:
Factory is not expected to be executed multiple times. So in fact, you can do this:

```suggestion
        [ck_args = std::move(ck_args), data_type, config_idx](const std::vector<Kernel>& kernels) mutable {
            std::ignore = kernels;
            return [ck_args = std::move(ck_args), data_type, config_idx](const Handle& handle,
```

Reply:
Thank you for the suggestion. That was the piece I was missing yesterday and found out about it when dealing with `non-copyable` objects from CK (see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2305#discussion_r1294994063). 
The change above is implemented in the latest commit 9034f3ea